pan-domain-node/test/panda.test.ts

```diff
@@ -0,0 +1,28 @@
+import { guardianValidation, AuthenticationStatus } from '../src/api';
+import { verifyUser } from '../src/panda';
+
+import { sampleCookie, sampleCookieWithoutMultifactor, sampleNonGuardianCookie, publicKey } from './fixtures';
+
+test("return invalid cookie if missing", () => {
+    expect(verifyUser(undefined, "", 0, guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
+});
+
+test("return invalid cookie for a malformed signature", () => {
+    const [data, signature] = sampleCookie.split(".");
+    const testCookie = data + ".1234";
+
+    expect(verifyUser(testCookie, publicKey, 0, guardianValidation).status).toBe(AuthenticationStatus.INVALID_COOKIE);
+});
+
+test("return expired", () => {
+    expect(verifyUser(sampleCookie, publicKey, 5678, guardianValidation).status).toBe(AuthenticationStatus.EXPIRED);
```

Comment:
could we remove the magic number?

Reply:
No. It's a test for expiry where the passed in timestamp (the time "now" when called in real code outside of test)

Reply:
Would it be worth using a `const` here to improve readability? `const someTimeInTheFuture = 5678`